### ISSUE TITLE
Support For GitLab(Self-Managed)

### ISSUE THREAD

**Issue by fadu-yongyeon-park** [AtomMaterialUI/a-file-icon-web#276]
**Is your feature request related to a problem? Please describe.**

- I want to apply extension in gitlab(self-managed)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mallowigi** [comment]
So in short, you want to be able to add the ability to add custom urls?

**Comment by ian-park-fad** [comment]
Yes, that's correct. Currently, we are using GitLab Self-Managed with a domain such as gitlab.example.com. Is there a plan to support custom URLs?

**Comment by mallowigi** [comment]
There wasn't, but I am taking notes, maybe in the future.

**Comment by IP-Developer** [comment]
+1 for this. We're using only self-managed resources.
